stdlib/printexc.mli

```diff
@@ -193,7 +193,7 @@ val backtrace_slots : raw_backtrace -> backtrace_slot array option
     recent function call, raise, or primitive [get_backtrace] call in
     the trace.
 
-    Some possible reasons for returning [None] are as follow:
+    Some possible reasons for returning [None] are as follows:
```

Comment:
Agreed: "as follows" is only needed to make the phrase sound more pompous.

But we are entering the territory of controversial here. What if the author of `backtrace_slots` actually intended it to sound more important and somewhat "american", if only to set the mood for the user? I wouldn't want to ruin that finely crafted impression.